cmd/cmd-run.go

```diff
@@ -175,7 +176,22 @@ func run(cmd *cobra.Command, _ []string) error {
 		<-c
 		os.Exit(1)
 	}()
-
+	repoInclude, _ := flag.GetString("repo-include")
```

Comment:
Please move this code block above setting up the signal, so that the code reads
parsing flags, setting up signal
and not
parsing flags, setting up signal, parsing some more flags.